Testing/Unit/PowerShell/Connection/Connect-DefenderHelper.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath "../../../../PowerShell/ScubaGear/Modules/Connection/ConnectHelpers.psm1") -Function 'Connect-DefenderHelper' -Force
+
+InModuleScope ConnectHelpers {
+    Describe -Tag 'Connection' -Name 'Connect-DefenderHelper' {
```

Comment:
I directly changed the branch with the proposed changes for this unit test.  This is closer to what I'd expect for 'whitebox' unit testing.  Notice I removed the definition of Connect-IPPSSession and only mocked the function.  I believe you where mocking the function defined directly above the mock definition.  We can discuss further if you like.  I'd expect (at some time - perhaps in future) all the unit tests to be more comprehensive similar to this one.